src/helpers/supportsLSPInterface/supportsLSPInterface.test.ts

```diff
@@ -0,0 +1,61 @@
+import { INTERFACE_IDS } from '@lukso/lsp-smart-contracts';
+import { expect } from 'chai';
+
+// types
+import { ERC165 } from '../../typechain';
+
+// utils
+import { supportsLSPInterface } from '../..';
+import { BytesLike, Provider, ZeroAddress } from 'ethers';
+
+describe('supportsLSPInterface', () => {
+    const mockERC165 = {
+        getAddress: async () => ZeroAddress,
+        supportsInterface: async (interfaceId: BytesLike) => {
+            if (interfaceId === INTERFACE_IDS.LSP0ERC725Account) {
+                return true;
+            }
+            return false;
+        },
+    };
+    const nonAddressableMockERC165 = {
+        supportsInterface: async (interfaceId: BytesLike) => {
+            if (interfaceId === INTERFACE_IDS.LSP0ERC725Account) {
+                return true;
+            }
+            return false;
+        },
+    };
+    const mockProvider = {} as Provider;
+
+    it('it should support LSP0 interface and return true', async () => {
+        expect(await supportsLSPInterface(INTERFACE_IDS.LSP0ERC725Account, mockERC165 as ERC165)).to
+            .be.true;
+    });
+
+    it("it shouldn't support LSP0 interface and return false", async () => {
```

Comment:
I am confused by the name of this test. It says `shouldn't support LSP0 interface`, but the interface ID used is LSP7